crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -676,8 +716,19 @@ impl SlidingSync {
 
 #[derive(Debug)]
 pub(super) struct SlidingSyncPositionMarkers {
+    /// An ephemeral position in the current stream, as received from the
+    /// previous /sync response, or `None` for the first request.
+    ///
+    /// Should not be persisted.
     pos: Option<String>,
+
+    /// Server-provided opaque token that remembers what the last timeline and
+    /// state events stored by the client were.
+    ///
+    /// If `None`, the server will send the
+    /// full information for all the lists present in the request.
     delta_token: Option<String>,
+
     to_device_token: Option<String>,
```

Comment:
It's great to add documentation for those fields, thanks!
Can you document `to_device_token` too please?